docs/grpc/docs.md

```
@@ -3384,6 +3386,28 @@ Additionally, the first and last points of each GeoLineString must be the same.
 
 
 
+<a name="qdrant-Mmr"></a>
+
+### Mmr
+Maximal Marginal Relevance (MMR) algorithm for re-ranking the points.
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| diversity | [float](#float) | optional | Tunable parameter for the MMR algorithm. Determines the balance between diversity and relevance.
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)